python/cugraph-pyg/cugraph_pyg/tests/test_cugraph_loader.py

```diff
@@ -201,6 +201,10 @@ def test_cugraph_loader_from_disk_subset():
 
 @pytest.mark.skipif(isinstance(torch, MissingModule), reason="torch not available")
 def test_cugraph_loader_from_disk_subset_csr():
+    # TODO: The CSC/CSR code path in cugraph_sampler.py requires torch_sparse
+    # as dependency. Remove this check when torch_sparse becomes a dependency.
+    pytest.importorskip("torch_sparse", reason="PyTorch Sparse not available")
```

Comment:
`importorskip` is not the RAPIDS-preferred way to handle skipping.  This should be changed to using `import_optional` and `pytest.mark.skipif` like we do for `torch`.